components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/contract-classes.adoc

```diff
@@ -20,7 +20,9 @@ A contract class does not necessarily have a deployed instance in Starknet.
 A contract class does not necessarily require a deployed instance in Starknet.
 ====
 
-== Working with Classes
+#A contract instance has a nonce, the value of which is the number of transactions originating from this address plus 1. For example, when you deploy an account with a `DEPLOY_ACCOUNT` transaction, the nonce of the account contract in the transaction is `0`. After the transaction, until the account contract sends its first transaction, the nonce is `1`.#
```

Comment:
```suggestion
#A contract instance has a nonce, the value of which is the number of transactions originating from this address plus 1. For example, when you deploy an account with a `DEPLOY_ACCOUNT` transaction, the nonce of the account contract in the transaction is `0`. After the deploy transaction, until the account contract invokes its next transaction, the nonce is `1`.#
```

Reply:
@odednaor these are the three transaction types:
![image](https://github.com/starknet-io/starknet-docs/assets/39279277/27395883-b150-40f6-b966-a41056e6d761)
So what do you mean by
> the deploy transaction
?

> until the account contract invokes its next transaction

`INVOKE` is a transaction type, so do you mean that the contract sends an `INVOKE` transaction? And what does it mean to invoke a transaction? You can send a transaction, you can invoke a function. I don't see how you can invoke a transaction.

Reply:
I meant the `DEPLOY_ACCOUNT` transaction.